### DISCUSSION TITLE
Running Docker Image with Sentinal Mode on Mac

### DISCUSSION THREAD

**Discussion by morgan-coxC21** [Grokzen/docker-redis-cluster#132] [Q&A]
Hello! I am trying to get this program to work with docker on a MacBook Pro. My goal is to run the cluster in sentinel mode locally, however I have limited docker experience and have not been able to find an answer to this question so far. 

From the readme if I am using a may I must specify:

- docker run -e "IP=0.0.0.0" -p 7000-7005:7000-7005 grokzen/redis/cluster:latest

when I do this everything works just fine but this is not running in sentiniel mode (checked by listing services at 5000-50002 ie lsof -i :5000 ) so, i have tried this:

- docker run -e "SENTINEL=true" -p 7000-7005:7000-7005 grokzen/redis/cluster:latest

However, this does not show anything on ports 5000-5002 but when I use lsof on ports for the master ones they are up there. 

So my question is, am I passing these command arguments correctly, should I be passing both since I am on a mac and want sentinel (if so how, I have tried a few ways and none work), or am I completely wrong on my thinking here?

Thank you!

**Comment by Grokzen** [comment]
You are probably missing to bind the ports from your host to the container because if you run sentinel mode you need to add `-p 5000-5002:5000-5002` to your docker run command as well. This is not specific for mac but this is a docker thing to pass the correct ports. I do see tho in the documentation that it is not super clear that you should add the sentinel ports to your command so that probably should be added in there to make it extra clear.
